recipes-kernel/linux/linux-mainline-rt/intel-nuc.cfg

```diff
@@ -0,0 +1,33 @@
+#
+# USB Host Controller Drivers
+#
+CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_PCI=y
+CONFIG_USB_XHCI_PLATFORM=y
```

Comment:
This configuration should be probably to linux-mainline-rt defconfig instead of dedicated fragment for Intel NUC